frontend/src/components/elements/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from "react"
+import { shallow } from "enzyme"
+import { Map as ImmutableMap } from "immutable"
+
+import Alert, { getAlertCSSClass } from "./Alert"
+import { Alert as AlertProto } from "autogen/proto"
+
+const getProps = (elementProps: object = {}): Props => ({
```

Comment:
Props interface is not defined, you could export interface Props from Alert.tsx and the import it from here